rust/psibase/src/main.rs

```diff
@@ -958,8 +958,7 @@ async fn get_package_registry(
 
 async fn boot(args: &BootArgs) -> Result<(), anyhow::Error> {
     let (client, _proxy) = build_client(&args.node_args.proxy).await?;
-    let now_plus_120secs = Utc::now() + Duration::seconds(120);
-    let expiration = TimePointSec::from(now_plus_120secs);
+    let expiration = TimePointSec::from(Utc::now() + Duration::seconds(240));
```

Comment:
tapos can only go up to 127 seconds in the future before it has to switch to the other form. We're not actually checking tapos during boot, though. The expiration serves no useful purpose during boot. Maybe we should just set it to zero and change transact to not check it.